src/vs/platform/telemetry/common/telemetryUtils.ts

```diff
@@ -80,6 +83,11 @@ export function isWelcomePageEnabled(storageService: IStorageService) {
 	return 'enableWelcomePage' in overrides ? overrides.enableWelcomePage : splitExperimentsRandomness(storageService).enableWelcomePage;
 }
 
+export function isShowTaskDocumentation(storageService: IStorageService) {
```

Comment:
This is not needed then (see below). The welcome page one (you probably looked at) was a special case I'm about to remove.